src/appshell/qml/Preferences/internal/AudioGenerationSection.qml

```diff
@@ -23,15 +23,26 @@ import QtQuick 2.15
 
 import MuseScore.Project 1.0
 import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
 
 BaseSection {
     id: root
 
-    title: qsTrc("appshell/preferences", "Generate MP3 audio for private cloud scores")
+    title: qsTrc("appshell/preferences", "Save to cloud")
 
     navigation.name: "AudioGenerationSettingsContent"
```

Comment:
Don't forget to change this name after https://github.com/mathesoncalum/MuseScore/pull/7/files#r1316876354